typescript/examples/vercel-ai-sdk-cdp-chatbot/chatbot.ts

```diff
@@ -0,0 +1,191 @@
+import {
```

Comment:
nit: The examples folder has grown quite a bit. Does not have to be this PR, but eventually we should create subfolders, and I think `langechain`/`vercel-ai-sdk` are appropriate groupings to organize examples under.